1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

2Ф (t_lambda) = lambda_

lambda_ = 0.95

In [42]:
t_lambda = 1.96

In [43]:
delta = 1.96 * 16 / (256 ** 0.5)
delta

1.96

Доверительный интервал

(М - delta, М + delta)

In [44]:
M = 80

In [45]:
M - delta

78.04

In [46]:
M + delta

81.96

Ответ: (78.04, 81.96)

2. В результате 10 независимых измерений некоторой величины X,выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [47]:
import numpy as np

In [48]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [49]:
len(a)

10

In [50]:
a_mean = a.mean()
a_mean

6.590000000000001

In [51]:
a_std = a.std(ddof=1)
a_std

0.4508017549014448

In [52]:
t_kr = 2.262

In [53]:
delta = t_kr * a_std / (10**0.5)
delta

0.32246174408757394

In [54]:
a_mean - delta

6.267538255912426

In [55]:
a_mean + delta

6.912461744087575

Ответ ДИ = (6.267, 6.912)

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

H0: M = 17

H1: M > 17

In [56]:
U_nabl = (17.5-17) * 10 / 2

In [57]:
U_nabl

2.5

In [58]:
alpha = 0.05
Ф = (1-2*alpha) / 2

In [59]:
Ф

0.45

In [60]:
u_kr = 1.645

In [61]:
U_nabl < u_kr

False

Ответ: отвергаем H0

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [62]:
a = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [63]:
a_mean = a.mean()
a_mean

198.5

In [64]:
a_std = a.std(ddof=1)

In [65]:
a_std

4.453463071962462

H0: M = 200

H1: M < 200

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

In [69]:
U_nabl = ((200 - a_mean) * (10 ** 0.5))/ a_std
U_nabl

1.0651074037450896

In [67]:
u_kr = 3.250

In [68]:
U_nabl < u_kr

True

Ответ: принимаем H0